src/nodes.coffee

```diff
@@ -2715,10 +2717,19 @@ exports.Existence = class Existence extends Base
     code = @expression.compile o, LEVEL_OP
     if @expression.unwrap() instanceof IdentifierLiteral and not o.scope.check code
       [cmp, cnj] = if @negated then ['===', '||'] else ['!==', '&&']
-      code = "typeof #{code} #{cmp} \"undefined\" #{cnj} #{code} #{cmp} null"
+      code = "typeof #{code} #{cmp} \"undefined\" #{cnj} #{code} #{cmp} #{@comparisonTarget}"
```

Comment:
The second half of this expression (`#{code} #{cmp} #{@comparisonTarget}`) is redundant if `@comparisonTarget is 'undefined'` -

```js
typeof foo === 'undefined' || foo === undefined
typeof foo !== 'undefined' && foo !== undefined
```